### ISSUE TITLE
English Language Copyediting

### ISSUE THREAD

**Issue by acrymble** [programminghistorian/jekyll#1381]
As we get ready to publish our style guide in English, I have been thinking about the best way to implement it. I don't want this to become a huge burden for anyone, but I think it is an important step we can take to increase the quality of our writing and reduce the need to spend so much time with authors working on their prose.

I would like to suggest the following workflow (assuming that #1301 is happening and we will soon be a legal entity).

1. Authors guidelines for English only are replaced with the style guide - ensuring no information needed for formatting is lost (Action: me)
2. Authors required to upload a new form with checkboxes saying they have adhered to each section of the style guide prior to reviewer assignment.
3. Upon Acceptance by Editor/Managing Editor, editor decides if copy editing is required. Authors have the option of either referring again to the style guide and doing it themselves, or paying a copyediting fee (which we will advertise and explain openly) of £200 for people based in G20 / EU countries, or £0 for everyone else.
4. A copyeditor paid to apply the style guide rigorously, as necessary.
5. Lesson checked over by editor (who has had to spend considerably less time on these tasks) and then published / revised as appropriate.

Any views on this? I think that puts the burden for good writing largely on the authors, where I think it should be, and offers a professional support if required. It should shift the work that's currently done by editors to someone else, since I think we're overstretched on this task.

**Comment by drjwbaker** [comment]
On 1., what part *specifically* of https://programminghistorian.org/en/author-guidelines are you suggesting be replaced with the Style Guide. I see this as an addition rather than a replacement.

On 3., we need to wait until we have money in the bank until we can commit to £0 non G20 / EU cost for employing a copyeditor. Based on lessons submitted over the last year, how much would this cost us annually?

**Comment by acrymble** [comment]
As I've outlined in the pull request for the style guide, I've integrated the new suggestions into the existing Author's Guidelines and reorganized the material to try to make it more user-friendly. The new Author Guideline page should make it much easier for authors to follow, so I think we should consider numbers 1 and 2 above resolved by that, until we see how it actually gets used.

**Comment by acrymble** [comment]
> On 3., we need to wait until we have money in the bank until we can commit to £0 non G20 / EU cost for employing a copyeditor. Based on lessons submitted over the last year, how much would this cost us annually?

£0 in the past year. But if we start translating lessons from Spanish into English that might go up. Likewise if our globalization efforts start to bear fruit.

**Comment by drjwbaker** [comment]
> > On 3., we need to wait until we have money in the bank until we can commit to £0 non G20 / EU cost for employing a copyeditor. Based on lessons submitted over the last year, how much would this cost us annually?
> 
> £0 in the past year. But if we start translating lessons from Spanish into English that might go up. Likewise if our globalization efforts start to bear fruit.

Okay.

**Comment by amsichani** [comment]
Hi @acrymble , once this is now integrated into the existing Author's Guidelines , shall we close the ticket?

**Comment by acrymble** [comment]
@amsichani can we discuss this at our next board meeting first?

**Comment by drjwbaker** [comment]
With apologies for being the nomenclature pedant around here, I presume you mean Editorial Board. We need to be clear to distinguish that from the Board of Directors so as to avoid confusion.

**Comment by acrymble** [comment]
We discussed this at the September 2019 meeting and we decided to hold on the idea of charging for copy editing while we discuss business model options. There was concerns about the possible arbitrary nature of the decision, and the potential for a surprise cost late in the workflow. It was acknowledged that copy editing does have a cost to the project and must  be accounted for.

We agreed that editors will make a concerted effort to ensure that authors follow the new style guidelines, and that managing editors are empowered to send lessons back that in their opinion to not substantially meet the spirit of the guidelines.
